build-support/bin/BUILD

```diff
@@ -20,6 +20,7 @@ python_tests(
 )
 
 pex_binaries(
+    execution_mode="venv",
```

Comment:
Ok, that was obvious in retrospect. In 2.14.x we use Josh's direct-run support w/o a `pex_binary` target and that uses a venv. Here we have to manually call the shot that we need maximum compatibility with standard venvs (avoid the PYTHONPATH propagation of PEX_EXTRA_SYS_PATH cascade of hacks).